proto/toolchain.bzl

```diff
@@ -29,9 +29,20 @@ def _emit_proto_compile(ctx, proto_toolchain, go_proto_toolchain, lib, importpat
       mnemonic = "GoProtocGen",
       executable = proto_toolchain.protoc,
       arguments = args,
+      #command = "tree",
   )
   return go_srcs
 
+def proto_path(proto):
```

Comment:
Add doc string. The body of the commit message is fine for this.

Also, can this be private?